src/Mvc/Mvc.Core/src/Infrastructure/DefaultOutputFormatterSelector.cs

```diff
@@ -149,8 +149,6 @@ public override IOutputFormatter SelectFormatter(OutputFormatterCanWriteContext
 
             if (selectedFormatter == null)
             {
-                // No formatter supports this.
-                _logger.NoFormatter(context);
```

Comment:
This resulted in the log message being repeated twice:

```
warn: Microsoft.AspNetCore.Mvc.Infrastructure.DefaultOutputFormatterSelector[1]
      No output formatter was found for content type '' to write the response.
warn: Microsoft.AspNetCore.Mvc.Infrastructure.ObjectResultExecutor[1]
      No output formatter was found for content type '' to write the response.
```

I opted to remove so `ObjectResultExecutor` would still log it, even if `DefaultOutputFormatterSelector` has been swapped with a different service.